### PR TITLE
Update Directions to v2.6.0-beta.2

### PR DESCRIPTION
**Pull request by S2Ler**
Updates project to use MapboxDirections v2.6.0-beta.2

### CODE REVIEW

File: MapboxCoreNavigation.podspec [hunk 1]
```diff
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
   # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
   s.name = "MapboxCoreNavigation"
-  s.version = '2.6.0-alpha.2'
+  s.version = '2.6.0-beta.1'
```
Comment (MaximAlien) [inline MapboxCoreNavigation.podspec:6]: Was this modified manually? Shouldn't we do Navigation SDK related updates in scope of https://github.com/mapbox/mapbox-navigation-ios/pull/3936?

File: Tests/SPMTest/CoreSPMTest/CoreSPMTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -0,0 +1,160 @@
+{
```
Comment (chezzdev) [inline Tests/SPMTest/CoreSPMTest/CoreSPMTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:1]: One more `Package.resolved` to update? 🙈 Does it make sense it add one?

**Review by chezzdev — APPROVED** [review]
Please consider updating `Package.resolved` stored in `SPM.xcodeproj`.

File: Tests/SPMTest/CoreSPMTest/CoreSPMTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 2 above)
Comment (chezzdev) [reply Tests/SPMTest/CoreSPMTest/CoreSPMTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved]: Nice, but we still have `-SPM.xcodeproj` to update, could you do it?

**Pull request merged**